sat/apiclient/jobstat.py

```diff
@@ -49,8 +49,8 @@ def get_all(self):
         err_prefix = 'Failed to get State Checker data'
         try:
             response = self.get('all').json()
-            if 'errorcode' in response and response['errorcode'] != 0:
-                raise APIError(response.get('errormessage', 'unknown error'))
+            if 'status' in response and response['status'] != 201:
```

Comment:
With the latest version of the state checker properly returning HTTP error codes, we no longer need these lines, could we just remove them entirely?

For other commands, the error checking is already done [here](https://github.com/Cray-HPE/sat/blob/integration/sat/apiclient/gateway.py#L169-L183). The only reason we needed these two lines originally was because the service was returning the "OK" HTTP responses even in error cases.

I don't really like that we are saying only 201 is valid anyway, I think if the service were to return 200 or 202 that should not cause an error.